index.js

```diff
@@ -19,5 +19,24 @@ module.exports = function(source) {
         this.cacheable();
     }
 
+    // if force sourcemap requested embed inline sources (helps resolve issues with
+    // transpiled languages like typescript
+    if (userOptions['force-sourcemap']) {
```

Comment:
Is there a reason this needs to be hidden behind a flag? Can't this simply be:

```js
if (sourceMap) {
    var encoded = new Buffer(JSON.stringify(sourceMap)).toString('base64');
    source += '\n\n//# sourceMappingURL=data:application/json;base64,' + encoded;
}
```

If someone wants to ignore source maps, they can use `exclude`.